tensorflow_datasets/image_classification/isic2019.py

```diff
@@ -0,0 +1,439 @@
+"""isic2019 dataset."""
```

Comment:
TFDS now supports and uses datasets in file system format consisting of the source code, test, url checksums and the dummy data in the dataset folder. 
Kindly refer to [this](https://www.tensorflow.org/datasets/add_dataset#write_your_dataset) for porting this dataset to the described format